tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbStorageTests.cs

```diff
@@ -407,6 +411,76 @@ public async Task WaterfallCosmos()
             }
         }
 
+        [TestMethod]
+        public async Task DeleteAsyncFromSingleCollection()
+        {
+            string documentId = "UtteranceLog-001";
+            string[] keys = { documentId };
+            var storage = new CosmosDbStorage(CreateCosmosDbStorageOptions());
+            var item = new StoreItem { MessageList = new string[] { "hi", "how are u" }, City = "Contoso" };
```

Comment:
I noticed that the creation of the `StoreItem` is repeated in `DeleteAsyncFromPartitionedCollection` & `DeleteAsyncFromPartitionedCollectionWithoutPartitionKey` methods. As this class is only used for adding to a `Dictionary` for storing items, can we create a `private` object for not being constantly instantiating this and repeating code??